### ISSUE TITLE
WET documentation pages that need to be completed

### ISSUE THREAD

**Issue by zachfalsetto** [wet-boew/wet-boew#5574]
* [x] Archiving content: http://wet-boew.github.io/v4.0-ci/docs/ref/archived/archived-en.html
* [x] Country content: http://wet-boew.github.io/v4.0-ci/docs/ref/country-content/country-content-en.html
* [x] Data ajax: http://wet-boew.github.io/v4.0-ci/docs/ref/data-ajax/data-ajax-en.html
* [x] Data inview: http://wet-boew.github.io/v4.0-ci/docs/ref/data-inview/data-inview-en.html
* [x] Menu: http://wet-boew.github.io/v4.0-ci/docs/ref/menu/menu-en.html
* [x] Multimedia player: http://wet-boew.github.io/v4.0-ci/docs/ref/multimedia/multimedia-en.html
* [x] Overlay: http://wet-boew.github.io/v4.0-ci/docs/ref/overlay/overlay-en.html
* [x] Tabbed interface: http://wet-boew.github.io/v4.0-ci/docs/ref/tabs/tabs-en.html
* [x] Tables: http://wet-boew.github.io/v4.0-ci/docs/ref/tables/tables-en.html
* [x] Twitter: http://wet-boew.github.io/v4.0-ci/docs/ref/twitter/twitter-en.html

**Comment by hsrudnicki** [comment]
Are you ready to close this issue or otherwise?

**Comment by LaurentGoderre** [comment]
Closing since all the documentation was done

**Comment by EricDunsworth** [comment]
@zachfalsetto @LaurentGoderre Should this issue be reopened?

There's still lots of missing/incomplete/untranslated documentation out there...

1. **Generic placeholder documentation:**
  * Table Validator
    * http://wet-boew.github.io/v4.0-ci/docs/ref/tablevalidator/tablevalidator-en.html
    * http://wet-boew.github.io/v4.0-ci/docs/ref/tablevalidator/tablevalidator-fr.html
  * Transitions
    * http://wet-boew.github.io/v4.0-ci/docs/ref/transitions/transitions-en.html
    * http://wet-boew.github.io/v4.0-ci/docs/ref/transitions/transitions-fr.html
2. **Commented-out "Use when"/"Do not use when" documentation:**
  * Data Ajax
    * http://wet-boew.github.io/v4.0-ci/docs/ref/data-ajax/data-ajax-en.html
    * http://wet-boew.github.io/v4.0-ci/docs/ref/data-ajax/data-ajax-fr.html
  * Overlay
    * http://wet-boew.github.io/v4.0-ci/docs/ref/overlay/overlay-en.html
    * http://wet-boew.github.io/v4.0-ci/docs/ref/overlay/overlay-fr.html
  * Tabbed Interface
    * http://wet-boew.github.io/v4.0-ci/docs/ref/tabs/tabs-en.html
    * http://wet-boew.github.io/v4.0-ci/docs/ref/tabs/tabs-fr.html
3. **Missing French translations:**
  * Countless pages (I got 110 hits for "needs translation" across 66 pages)

**Comment by LaurentGoderre** [comment]
Sure

**Comment by hsrudnicki** [comment]
Made "Going Stale" but please keep this issue's status updated whether this issue can be closed or otherwise? Thx

**Comment by hsrudnicki** [comment]
@masterbee and others:  A discussion as well as work effort had been started in regards to the WET-based docs and is ongoing, therefore in parallel please update this issue's status as well, whether this issue can be closed as resolved or otherwise, thank you

**Comment by hsrudnicki** [comment]
@masterbee and others: Additionally to the (my) last comment, here are the other issues: 6230, 6232 to take into consideration in regards to this documentation issue.

**Comment by hsrudnicki** [comment]
Is this still an issue with the backlog (or uplanned) milestone?  Please update the status/milestone of this for this current year, whether this can be closed or otherwise, thank you

**Comment by hsrudnicki** [comment]
As well Rob's comment of the 7572 issue  (No accents in FormValidation) to be included in the docs

**Comment by RobJohnston** [comment]
It was https://github.com/wet-boew/wet-boew/issues/7592#issuecomment-240710793, not 7572.

Even better is to filter by [is:open label:"Content: Documentation" ] (https://github.com/wet-boew/wet-boew/labels/Content%3A%20Documentation).

**Comment by hsrudnicki** [comment]
as well into consideration #6577 (Calendar of Events: WET3 code in WET4 documentation)

**Comment by hsrudnicki** [comment]
also #6646 (Change the location of the browser support documentation) for your further consideration to be included into the docs

**Comment by hsrudnicki** [comment]
another issue #7382 asking to be included/into consideration

**Comment by hsrudnicki** [comment]
a next more: #5557

**Comment by hsrudnicki** [comment]
as well this issue #7221 to be included

**Comment by hsrudnicki** [comment]
also #7405

**Comment by hsrudnicki** [comment]
if Timeout Session, then see the help questions raised in #7654 into consideration

**Comment by hsrudnicki** [comment]
Also #6230 wants to be included into the documentation

**Comment by hsrudnicki** [comment]
#6245 wants to be in the docs

**Comment by hsrudnicki** [comment]
maybe #6567
